core/src/test/kotlin/org/evomaster/core/database/extract/postgres/Ind0ExtractTest.kt

```diff
@@ -44,4 +45,14 @@ class Ind0ExtractTest : ExtractTestBasePostgres() {
 
         //TODO check the 3 views and all constraints
     }
+
+    @Test
+    fun testSqlInsert() {
+        val schema = SchemaExtractor.extract(connection)
+
+        val builder = SqlInsertBuilder(schema)
+//        val actions = builder.createSqlInsertionAction("x", setOf("p_at", "created_at"))
+//        val genes = actions[0].seeGenes()
+
```

Comment:
a new test, but commented out and no assertions?

Reply:
This is the test that creates the actions for the industrial case study. The schema is already extracted, but there are many column types (xml, jsonb, text) that are not currently supported by EvoMaster. How should I proceed on cases like this one? (i.e. I have the test case but is failing) Should I add an @ignore ?

Reply:
just add a brief "TODO" comment :)